vip-cache-manager/vip-cache-manager.php

```diff
@@ -212,6 +212,10 @@ public function execute_purges() {
 			return;
 		}
 
+		if ( defined( 'VIP_GO_PHPUNIT_RUNNING' ) && VIP_GO_PHPUNIT_RUNNING ) {
```

Comment:
Alternatively, would it work to put something like this into the bootstrap file?

```
remove_action( 'init', array( 'VIP_Cache_Manager', 'init' ) );
```